src/Layouts/Main.purs

```diff
@@ -5,20 +5,19 @@ import Prelude
 import Bootstrap.Layout as L
 import Components.About as CAbout
 import Components.Home as CHome
-import Control.Monad.Aff (Aff, launchAff_)
-import Control.Monad.Eff (Eff)
 import Data.Either.Nested (Either4)
 import Data.Functor.Coproduct.Nested (Coproduct4)
 import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import Effect.Aff (Aff, launchAff_)
```

Comment:
my main inspiration for this is the work from @garyb here: https://github.com/slamdata/purescript-halogen/commit/d95b01f15d12d35661acf4b8abf496a37da10c13